app/services/google_search_manager/google_search_fetcher.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'selenium-webdriver'
+
+module GoogleSearchManager
+  class GoogleSearchFetcher < ApplicationService
+    def initialize(keyword)
+      @keyword = keyword
+    end
+
+    def call
+      url = "https://www.google.com/search?q=#{CGI.escape(@keyword.name)}"
```

Comment:
Prefer defining this as a constant:

```ruby
class GoogleSearchFetcher < ApplicationService
   GOOGLE_SEARCH_URL = "https://www.google.com/search?q="

   #...

    def call
       url = "#{GOOGLE_SEARCH_URL}#{CGI.escape(@keyword.name)}"
```

Avoid inlining hardcoded values to make the code easier to read.